syntax_and_semantics/if_var.md

```diff
@@ -33,30 +33,45 @@ Here, the right-hand side of the `&&` expression is also guaranteed to have `a`
 
 Of course, reassigning a variable inside the `then` branch makes that variable have a new type based on the expression assigned.
 
-The above logic **doesn’t** work with instance variables or class variables:
+## Variables with Influence
```

Comment:
I'm not pleased with this title but couldn't find a good one. If there are any suggestions...?

Reply:
Just call the section "Limitations"